blocked-edges/4.13.0-PerformanceProfilesCPUQuota.yaml

```diff
@@ -0,0 +1,13 @@
+to: 4.13.0
+from: 4[.]12[.].*
+url: https://issues.redhat.com/browse/OCPNODE-1705
+name: PerformanceProfilesCPUQuota
+message: |-
+  Clusters using PerformanceProfiles may have CPU quota issues on vulnerable 4.13.
```

Comment:
I'd consider being less vague about the observable impact and maybe not mention "vulnerable 4.13" which is kinda implied?

How about something like "Pods may not execute properly because of CPU quota issues on clusters using PerformanceProfiles"?

Reply:
Works for me; updated in 62262842 -> b5738722.